src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -1756,26 +1829,51 @@
 helps['network vnet subnet delete'] = """
     type: command
     short-summary: Delete a subnet.
+    long-summary: To learn more about deleting a virtual network visit https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-manage-subnet#delete-subnet
+    examples:
+        - name: Delete a subnet
+          text: az network vnet subnet delete -n mySubnet -g MyResourceGroup
 """
 
 helps['network vnet subnet list'] = """
     type: command
     short-summary: List subnets.
+    long-summary: To learn more about subnets in Azure, visit https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-manage-subnet
+    examples:
+        - name: Typical usage
+          text: az network vnet subnet list -n myVNet -g myResourceGroup
 """
 
 helps['network vnet subnet show'] = """
     type: command
     short-summary: Show details of a subnet.
+    long-summary: To learn more about subnets, visit https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-manage-subnet
+    examples:
+        - name: Show subnets associated to a specific virtual network
+          text: az network vnet subnet show -n mySubnet– vnet-name myVNet -g myResourceGroup
 """
 
 helps['network vnet subnet update'] = """
     type: command
     short-summary: Update a subnet.
+    long-summary: To learn more about subnets, visit https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-manage-subnet
     parameters:
         - name: --service-endpoints
           short-summary: Space separated list of services allowed private access to this subnet.
           populator-commands:
             - az network vnet list-endpoint-services
+    examples:
```

Comment:
Improve example. Suggestion: Include more arguments.

Reply:
Rec: Show a generic update example.

Reply:
Left, for now.